### PR TITLE
Make example consistent across related articles

### PR DESCRIPTION
**Pull request by Ana-AlonsoCanizares**
Make example consistent with that of .create ingestion mapping article

### CODE REVIEW

**Comment by American-Dipper** [comment]
@shsagir - can you review the proposed changes?  

 
IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge.  

 

#label:"aq-pr-triaged" 

@MicrosoftDocs/public-repo-pr-review-team

**Comment by shsagir** [comment]
@Ana-AlonsoCanizares Thanks for contacting us with feedback about our product documentation. Depending on the complexity, it could take from a few business days to a few weeks to look further into your question, issue, or suggestion. We'll contact you if we need further information, and you'll receive an email message each time we "@mention" you in a comment. 

Thanks again for submitting your feedback. Your suggestions help improve our documentation.

File: data-explorer/kusto/management/alter-ingestion-mapping-command.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Alters an existing ingestion mapping that is associated with a specific table/da
 .alter table MyTable ingestion json mapping "Mapping1"
 ```
 [
-    { "column" : "rownumber", "Properties":{"Path":"$.rownumber"}},
+    { "column" : "rownumber", "datatype" : "int", "Properties":{"Path":"$.rownumber"}},
     { "column" : "rowguid", "Properties":{"Path":"$.rowguid"}}
```
Comment (sigorbor) [inline data-explorer/kusto/management/alter-ingestion-mapping-command.md:45]: Need to add "DataType":"string" for the rowguid

Comment (shsagir) [inline data-explorer/kusto/management/alter-ingestion-mapping-command.md:45]:
```suggestion
    { "column" : "rowguid", "DataType":"string", "Properties":{"Path":"$.rowguid"}}
```

File: data-explorer/kusto/management/alter-ingestion-mapping-command.md [hunk 2]
```diff
@@ -41,8 +41,8 @@ Alters an existing ingestion mapping that is associated with a specific table/da
 .alter table MyTable ingestion json mapping "Mapping1"
 ```
 [
-    { "column" : "rownumber", "Properties":{"Path":"$.rownumber"}},
-    { "column" : "rowguid", "Properties":{"Path":"$.rowguid"}}
+    { "column" : "rownumber", "datatype" : "int", "Properties":{"Path":"$.rownumber"}},
```
Comment (shsagir) [inline data-explorer/kusto/management/alter-ingestion-mapping-command.md:44]:
```suggestion
    { "column" : "rownumber", "DataType" : "int", "Properties":{"Path":"$.rownumber"}},
```

**Comment by shsagir** [comment]
@Ana-AlonsoCanizares Thank you for your contribution. We've submitted a documentation change based on your suggestion, and the updated article should be live by tomorrow.

#sign-off

**Pull request merged**